ComponentKit/LayoutComponents/CKBackgroundLayoutComponent.mm

```diff
@@ -40,6 +40,11 @@ + (instancetype)newWithView:(const CKComponentViewConfiguration &)view size:(con
   CK_NOT_DESIGNATED_INITIALIZER();
 }
 
+- (std::vector<CKComponent *>)render:(id)state
+{
+  return {_component, _background};
```

Comment:
how does the infrastructure know to apply the same logic that is currently in `computeLayoutThatFits`?

Reply:
The layout process is the same as it was before. The render method only being called in order to construct the component tree (`CKComponentTreeNode`). In the future, we could unify the two.